app/presenters/publishing_api_presenters/political_helper.rb

```diff
@@ -0,0 +1,12 @@
+module PublishingApiPresenters::PoliticalHelper
+private
+
+  def government
+    gov = item.government
+    {
+      title: gov.name,
+      slug: gov.slug,
+      current: gov.current?
+    }
```

Comment:
This line is also associated with something being political:
https://github.com/alphagov/whitehall/blob/99034225d518705c56fb6b42c1a8a273854c1d5b/app/presenters/publishing_api_presenters/detailed_guide.rb#L32

I feel like they should somehow be kept together.